snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: plexmediaserver
-version: 1.2.2.2857-d34b464-1
+version: 1.2.2.2857-d34b464
```

Comment:
I've rolled it back but I disagree with the idea of doing this so I will keep my own fork.